### PR TITLE
from trillion fork to usdoj main repo

### PR DESCRIPTION
**Pull request by harikrishnagit**
[Link to ZenHub issue.](link-goes-here)

### Author

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] Check for [accessibility](/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,35 +29,37 @@
...
+    "gulp-sourcemaps": "^2.6.5",
     "node-sass": "^4.13.1",
     "pa11y": "^5.3.0",
     "pa11y-ci": "^2.3.0",
-    "prettier": "1.18.2"
+    "postcss-csso": "^4.0.0",
+    "prettier": "1.18.2",
+    "sass": "^1.26.9",
+    "uswds": "^2.7.1",
+    "uswds-gulp": "github:uswds/uswds-gulp#5e32f1b99279a6e50ad875e66cc4cb77b456c390"
```
Comment (cantsin) [inline package.json:63]: I think we only want this one line to be changed (and its corresponding equivalent in `package-lock.json`

**Comment by Jkrzy** [comment]
Hi @harikrishnagit, it would be great to get this one into the `develop` branch ASAP to help others as they roll onto the project.

File: package.json (hunk 1 above)
Comment (harikrishnagit) [reply package.json:63]: Ok, will do.

**Comment by harikrishnagit** [comment]
Fixed merge conflict.

**Pull request merged**